docs/plugins/subsonicupdate.rst

```diff
@@ -12,6 +12,12 @@ Then, you'll probably want to configure the specifics of your Subsonic server.
 You can do that using a ``subsonic:`` section in your ``config.yaml``,
 which looks like this::
 
+    subsonic:
+        url: https://mydomain.com:443/subsonic
+        user: username
+        pass: password
+
+    # DEPRECATED
```

Comment:
Let's just delete the deprecated configuration stuff altogether. Nobody needs to know it exists—they can just keep using it if it's already in their config file. :smiley: